unit/testing-utils/load_java_class.cpp

```diff
@@ -109,3 +132,18 @@ symbol_tablet load_java_class(
   REQUIRE_FALSE(class_type.get_bool(ID_incomplete_class));
   return std::move(maybe_goto_model->symbol_table);
 }
+
+symbol_tablet load_java_class(const std::string &java_class_name,
+                              const std::string &class_path,
+                              const std::string &main,
+                              std::unique_ptr<languaget> &&java_lang)
+{
+  cmdlinet command_line;
+  // TODO(tkiley): This doesn't do anything as "java-cp-include-files" is an
+  // TODO(tkiley): unknown argument. This could be changed by using the
+  // TODO(tkiley): free_form_cmdlinet
+  command_line.set("java-cp-include-files", class_path);
```

Comment:
Fixing this does indeed cause problems - I will leave as it is for now and raise a ticket to address

Reply:
I understand it now ;)